pkg/machine/ignition.go

```diff
@@ -89,6 +90,24 @@ ExecStart=/bin/sh -c '/usr/bin/echo Ready >/dev/%s'
 [Install]
 RequiredBy=multi-user.target
 `
+	deMoby := `[Unit]
+Description=Remove moby-engine, symlink docker to podman
+# Run once for the machine
+After=systemd-machine-id-commit.service
+Before=zincati.service
+ConditionPathExists=!/var/lib/%N.stamp
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/rpm-ostree override remove moby-engine
+ExecStart=/bin/ln -fs /usr/bin/podman /usr/local/bin/docker
```

Comment:
```suggestion
```

Thinking on this a bit more let's also create this symlink using Ignition and not in this systemd unit unless there's a good reason not to.

Reply:
if we do move the symlink to ignition then we should change the description of this unit (again).

Reply:
Yeah I agree. In fact, I *almost* did that with the other fixup, but hesitated thinking I should probably just keep changes limited to the feedback.